x/crosschain/keeper/evm_hooks_test.go

```diff
@@ -169,9 +170,16 @@ func TestValidateZrc20WithdrawEvent(t *testing.T) {
 			*sample.GetValidZRC20WithdrawToBTC(t).Logs[3],
 		)
 		require.NoError(t, err)
-		btcMainNetWithdrawalEvent.Value = big.NewInt(0)
+
+		// 1000 satoshis is the minimum amount that can be withdrawn
```

Comment:
maybe we can set the test name to 
`unable to validate a btc withdrawal event with an invalid amount`

Reply:
https://github.com/zeta-chain/node/pull/2362/commits/28973eaa3086ba0cbfafcddf5ad78c09efa9386b